egs2/TEMPLATE/svs1/scripts/utils/copy_data_dir.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
Another option is to still use Kaldi's one, but at the same time, setting utt_extra_files like https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L609-L621

Reply:
Label and xml.scp are added into utt_extra_files in svs.sh.